managed-token/sdk/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@jarry-xiao I just noticed (too late) that the dependencies here need some changes.

- `rustbin` + `solita` should be `devDependencies`
- deps that the generated code is using should be added as well

Extra deps needed:

- @metaplex-foundation/beet
- @solana/web3.js

Maybe we can fix this in a separate PR.

Reply:
ah sad, i can potentially look into adding. Could that all be generated with shank/solita or are there issues there?

Reply:
I've thought of generating this, but don't want to mess with the user's `package.json` file.
Specifically they might already have a `web3.js` dep in there for instance and I wouldn't want to change that and/or add a duplicate.

However if we come up with a frictionless way to do this for users I'm open to add this as a feature to solita.